src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -195,9 +195,22 @@
               </form>
               <div *ngIf="isPublic">
                 <span><strong matTooltip="Short description of the workflow">Topic</strong>:&nbsp;</span>
-                <span>{{
-                  workflow?.topicSelection === TopicSelectionEnum.AUTOMATIC ? workflow?.topicAutomatic : workflow?.topicManual
-                }}</span>
+                <span *ngIf="workflow?.topicSelection === TopicSelectionEnum.AUTOMATIC">{{ workflow?.topicAutomatic }}</span>
+                <span *ngIf="workflow?.topicSelection === TopicSelectionEnum.MANUAL">{{ workflow?.topicManual }}</span>
+                <span *ngIf="workflow?.topicSelection === TopicSelectionEnum.AI" data-cy="topic-selected-ai-text">
+                  {{ workflow?.topicAI }}
+                  <a
+                    [href]="Dockstore.DOCUMENTATION_URL + '/faq.html'"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    data-cy="ai-bubble"
+                    class="bubble ai-bubble"
+                    matTooltip="This topic sentence was AI generated. Non-published workflows will not be sent for AI topic sentences."
```

Comment:
This always is a published workflow, isn't it, based on line 196? If so, I think it's confusing to have ` Non-published workflows will not be sent for AI topic sentences.` -- I'd just drop it.